packages/es-components/src/components/base/spinner/Spinner.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const rotatorAnimation = keyframes`
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(270deg);
+  }
+`;
+
+const dashAnimation = keyframes`
+  0% {
+    stroke-dashoffset: 187;
+  }
+
+  50% {
+    stroke-dashoffset: 37.4;
+    transform: rotate(135deg);
+  }
+
+  100% {
+    stroke-dashoffset: 187;
+    transform: rotate(450deg);
+  }
+`;
+
+const colorsAnimation = keyframes`
+  0% {
+    stroke: #00a0d2;
+  }
+  40% {
+    stroke: #00c389;
+  }
+  80% {
+    stroke: #c110a0;
+  }
+  100% {
+    stroke: #00a0d2;
```

Comment:
How many of these colors can we pull from `theme`?

Reply:
@Darrken This might yet be another thing to worry about in a separate PR if need be.

Reply:
We may be able to grab them off the theme, and we might want to. This was just copied over from the WTW Style Guide.

Reply:
If these are colors that we'll use across other components, it should go in the theme. Otherwise, this is fine to remain as is, in my opinion.

Reply:
I know at least the `#00c389` used by the `40%` keyframe is `vbGreen` in the wtw theme. I can easily see coloring being different between wtw theme, via theme and any other themes that eventually come down the line. However, that is where I think that's a problem to think about for another time. It's good as is. If we run into issues with projects using the wtw theme and the coloring here being off we can worry about it then.